baseplate/secrets/store.py

```diff
@@ -306,18 +293,20 @@ def make_object_for_context(self, name, span):
 class _CachingSecretsStore(SecretsStore):
     """Lazily load and cache the parsed data until the server span ends."""
 
-    def __init__(self, filewatcher):  # pylint: disable=super-init-not-called
+    def __init__(self, filewatcher: FileWatcher):  # pylint: disable=super-init-not-called
         self._filewatcher = filewatcher
 
     @cached_property
     def _data(self):
-        return super(_CachingSecretsStore, self)._get_data()
+        return super()._get_data()
 
-    def _get_data(self):
+    def _get_data(self) -> Dict:
```

Comment:
I'm guessing we don't know anything about what's in this dictionary?

Reply:
yeah, we could get a bit more precise in that the top level will be str->dict but then that underlying dict will be a lot more undefined. not sure how precise we need to be.